### PR TITLE
New version: BornToBeRoot.NETworkManager version 2022.2.22

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/47706)

### CODE REVIEW

**Review by ItzLevvie — DISMISSED** [review]

File: manifests/b/BornToBeRoot/NETworkManager/2022.2.22/BornToBeRoot.NETworkManager.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2022-02-22
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/BornToBeRoot/NETworkManager/releases/download/2022.2.22/NETworkManager_2022.2.22.0_Setup.exe
```
Comment (BornToBeRoot) [inline manifests/b/BornToBeRoot/NETworkManager/2022.2.22/BornToBeRoot.NETworkManager.installer.yaml:20]: URL changed to https://github.com/BornToBeRoot/NETworkManager/releases/download/2022.2.22.0/NETworkManager_2022.2.22.0_Setup.exe

File: manifests/b/BornToBeRoot/NETworkManager/2022.2.22/BornToBeRoot.NETworkManager.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: BornToBeRoot.NETworkManager
+PackageVersion: 2022.2.22
```
Comment (BornToBeRoot) [inline manifests/b/BornToBeRoot/NETworkManager/2022.2.22/BornToBeRoot.NETworkManager.yaml:5]: Version changed to 2022.2.22.0

File: manifests/b/BornToBeRoot/NETworkManager/2022.2.22/BornToBeRoot.NETworkManager.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+- lldp
+- pingmonitor
+- portscanner
+- putty
+- remotedesktop
+- traceroute
+- wifi
+# Agreements: 
+# ReleaseNotes: 
+ReleaseNotesUrl: https://github.com/BornToBeRoot/NETworkManager/releases/tag/2022.2.22
```
Comment (BornToBeRoot) [inline manifests/b/BornToBeRoot/NETworkManager/2022.2.22/BornToBeRoot.NETworkManager.locale.en-US.yaml:33]: Url changed to https://github.com/BornToBeRoot/NETworkManager/releases/tag/2022.2.22.0

File: manifests/b/BornToBeRoot/NETworkManager/2022.2.22/BornToBeRoot.NETworkManager.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: BornToBeRoot.NETworkManager
+PackageVersion: 2022.2.22
```
Comment (BornToBeRoot) [inline manifests/b/BornToBeRoot/NETworkManager/2022.2.22/BornToBeRoot.NETworkManager.installer.yaml:5]: 2022.2.22.0

**Comment by BornToBeRoot** [comment]
Hi @vedantmgoyal2009 

i created a wrong tag on my repo. The tag changed from `2022.2.22` to `2022.2.22.0` but the version / url here is still wrong.

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**